apps/chrome-devtools/package.json

```diff
@@ -12,6 +12,7 @@
     "build": "yarn nx build chrome-devtools",
     "postbuild:patch": "yarn patch:manifest && yarn patch:extension && yarn patch:package",
     "patch:package": "cpy 'package.json' 'dist' && patch-package-json-main",
+    "copy:assets": "cpy 'src/manifest.json' 'src/devtools.html' dist --flat && cpy './src/assets/**' dist",
```

Comment:
why it is now needed?

Reply:
Because we cannot do it through assets anymore - the `output` cannot be outside the root (no `../`) so I created a new script and nx task instead.